src/main/java/com/userservice/Controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package com.userservice.Controller;
+
+import com.userservice.Entity.Users;
+import com.userservice.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+@RestController
+@RequestMapping("/api")
+public class UserController {
+    @Autowired
+    private UserService userService;
+    @PostMapping("/create")
+    public ResponseEntity<?>createUser(@RequestBody Users users){
+        return userService.create(users);
+    }
+    @GetMapping("getUser/{username}")
+    public ResponseEntity<?>getUserDetails(@PathVariable String username){
+      return  userService.getUserDetails(username);
+    }
+    @DeleteMapping("/delete/{username}")
```

Comment:
Delete with phone Number (phone number and country code), not with the username.

Reply:
Also, create the function to delete the user with ID too